soh/src/code/z_play.c

```diff
@@ -870,11 +885,9 @@ void Play_Update(PlayState* play) {
                                 R_UPDATE_RATE = 3;
                             }
 
-                            // Autosave on area transition unless you're in a cutscene or a grotto since resuming from grottos breaks the game
```

Comment:
I KNOW this isn't an intentional change since I made this change to update autosave to use an enum instead of a bool, so you definitely need to update your branch.

Reply:
I have no clue what happened, as I had just merged my branch last night after doing everything else, as you can see in the commits. But I will fix it.

Reply:
I took the code from the latest develop and replaced it with mine and just added back in what I was originally adding, So if there are still issues, I'm not sure what I'm supposed to do. I also made sure I was up to date with develop.